APMrover2/sailboat.h

```diff
@@ -68,14 +85,28 @@ class Sailboat
     AP_Float sail_angle_ideal;
     AP_Float sail_heel_angle_max;
     AP_Float sail_no_go;
+    AP_Float max_cross_track;
+    AP_Float loit_radius;
```

Comment:
is the extra loiter radius really necessary?  I guess the point is that a user might want to have different radius depending upon whether they are using a motor or are relying on just the sail.. it seems a bit of a luxury to me..

Reply:
Yeah that was the idea, we can use the normal loiter rad for the motor loiter and the new sailboat one for a sailing loiter. Not much point in using the motor in a loiter if you still have to have the big radius you need for the sailing loiter. Now we have a enable param I feel less bad about adding in more sailing stuff.